pkg/kubelet/volumemanager/cache/actual_state_of_world.go

```diff
@@ -550,30 +546,17 @@ func (asw *actualStateOfWorld) AddPodToVolume(markVolumeOpts operationexecutor.M
 	return nil
 }
 
-func (asw *actualStateOfWorld) MarkVolumeAsResized(
-	podName volumetypes.UniquePodName,
-	volumeName v1.UniqueVolumeName) error {
+func (asw *actualStateOfWorld) MarkVolumeAsResized(volumeName v1.UniqueVolumeName, claimSize *resource.Quantity) bool {
```

Comment:
why return bool instead error?

Reply:
there is no error to return here. Either volume was there in ASOW and was correctly marked as resized or it wasn't and nothing was recorded. In later case - if volume is no longer in ASOW, expansion won't be retried and error state will correct itself.